java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -452,16 +452,15 @@ CompletableFuture<String> xgroupCreate(
      *     Map</code> is composed of a stream's key and the id of the entry after which the stream
      *     will be read. Use the special id of <code>{@literal ">"}</code> to receive only new messages.
      * @param group The consumer group name.
-     * @param consumer The newly created consumer.
+     * @param consumer The consumer name.
```

Comment:
Can you please add this change to the other xreadgroup function signature as well?